### PR TITLE
[Settings]: fix mail icon on windows

### LINKED ISSUE #13259: On Windows, with Chrome and FireFox browsers, icon for Invite new user is broken
### Describe the bug

When SuperAdmin user goes through Settings > Users > Invite new user, after filling all the fields and press [Invite], the icon on the left of URL to copy to give access to new user is broken : 

### Steps to reproduce the behavior

1. Go to Settings > Users > Invite new user
2. After filling all the fields and press [Invite]
3. Pop-up appears with URL to send to new user for registration
4. See error : icon is broken (see screenshot below)

### Expected behavior

Icon should be complete.

### Screenshots

image

Jira card : https://strapi-inc.atlassian.net/browse/CONTENT-153

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Just use a span instead of an SVG (literally no point for this)

### Why is it needed?

* Windows has a problem with how that SVG is written

### How to test it?

* You'll need windows....

### Related issue(s)/PR(s)

* resolves #13259

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js [hunk 1]
```diff
@@ -46,7 +29,7 @@ const MagicLinkWrapper = ({ children, target }) => {
       title={target}
       titleEllipsis
       subtitle={children}
-      icon={<Envelope />}
+      icon={<span style={{ fontSize: 32 }}>✉️</span>}
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js:32]: wondering, why don't we use the DS icon? 👀
```suggestion
      icon={Envelop}
```

**Review by ronronscelestes — COMMENTED** [review]
wondering

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js (hunk 1 above)
Comment (joshuaellis) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js:32]: It's not the same icon! So wasn't sure whether to do that or not? 👀

Comment (ronronscelestes) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js:32]: ah right my bad, well still feel it's a shame we don't use DS icons but that could be another discussion!

**Review by ronronscelestes — APPROVED** [review]

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js (hunk 1 above)
Comment (joshuaellis) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js:32]: Something to add to our backlog?

Comment (ronronscelestes) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js:32]: yup 🚀

**Pull request merged**